VenueNextSDK-Demo.xcodeproj/project.pbxproj

```diff
@@ -317,18 +298,19 @@
 		AB6E2342228DCA2D00EDD290 /* Frameworks */ = {
 			isa = PBXGroup;
 			children = (
-				328C976D238EE60700667535 /* VNAnalytics.framework */,
-				328C9769238EE60600667535 /* VNCore.framework */,
-				328C9768238EE60600667535 /* VNCoreUI.framework */,
-				328C976B238EE60600667535 /* VNLegacy.framework */,
-				328C976C238EE60600667535 /* VNNetworkService.framework */,
-				328C976A238EE60600667535 /* VNOrderData.framework */,
-				328C9767238EE60600667535 /* VNOrderService.framework */,
-				328C9765238EE60500667535 /* VNOrderUI.framework */,
-				328C9766238EE60600667535 /* VNPayment.framework */,
-				328C976E238EE60700667535 /* VNWalletData.framework */,
-				328C9763238EE60500667535 /* VNWalletService.framework */,
-				328C9764238EE60500667535 /* VNWalletUI.framework */,
+				CE52A4A523BFB9DE0019B1F5 /* DynamiContent.framework */,
```

Comment:
In the Demo app we usually don't use DynamiContent. I don't think it hurts for this framework to be here, but we can remove it.